docs/docs/user-guide/getting-started/using-slurm.md

```diff
@@ -0,0 +1,339 @@
+
+#### **Example 2: Canceling all of your jobs**
+
+```bash
+scancel -u $USER
+```
+
+Cancels all jobs belonging to the current user.
+
+## 2. Example SBATCH script
+
+The following SBATCH script runs a BioNeMo Evo 2 training job inside a containerized environment.
+
+We'll walk through the script in detail below.
+
+```bash
+#!/bin/bash
+# SLURM directives
+# =========================
+#SBATCH --account=[INSERT ACCOUNT HERE]
+#SBATCH --nodes=1
+#SBATCH --partition=[INSERT PARTITIONs HERE]
+#SBATCH --ntasks-per-node=8
+#SBATCH --time=00:15:00
+#SBATCH --mem=0
+#SBATCH --job-name=[INSERT JOB NAME HERE]
+#SBATCH --mail-type=FAIL
+#SBATCH --exclusive
+set -x # Enable debugging (prints executed commands)
+# Job-specific parameters
+# =========================
+IMAGE_NAME=[INSERT IMAGE NAME HERE]
+EXPERIMENT_NAME=[INSERT PROJECT NAME HERE]
+MODEL_SIZE=7b
+CP_SIZE=1
+TP_SIZE=1
+PP_SIZE=1
+MICRO_BATCH_SIZE=2
+GRAD_ACC_BATCHES=1
+SEQ_LEN=8192
+MAX_STEPS=100
+VAL_CHECK=50
+CLIP_GRAD=250
+EXTRA_ARGS="--enable-preemption --use-megatron-comm-overlap-llama3-8k --ckpt-async-save --overlap-grad-reduce --clip-grad $CLIP_GRAD --eod-pad-in-loss-mask"
+EXTRA_ARG_DESC="BF16_perf_cg250_continue"
+LR=0.0003
+MIN_LR=0.00003
+WU_STEPS=2500
+# 0xDEADBEEF
+SEED=1234
+WD=0.1
+ADO=0.01
+HDO=0.01
+# Mounts
+# =========================
+DATA_PATH=/lustre/.../[INSERT DATA PATH HERE]
+DATA_MOUNT=/workspace/bionemo2/data
+MODEL_PATH=/lustre/.../[INSERT MODEL PATH HERE]
+MODEL_MOUNT=/workspace/bionemo2/model
+RESULTS_PATH=$MODEL_PATH/experiments/${EXPERIMENT_NAME}
+mkdir -p $RESULTS_PATH
+MOUNTS=${DATA_PATH}:${DATA_MOUNT},${MODEL_PATH}:${MODEL_MOUNT},$HOME/.cache:/root/.cache
+# Training command
+# =========================
+read -r -d '' COMMAND <<EOF
+echo "*******STARTING********" \
+&& echo "---------------" \
+&& echo "Starting training" \
+&&  \
+python /workspace/bionemo2/sub-packages/bionemo-evo2/src/bionemo/evo2/run/train.py \
+    -d /workspace/bionemo2/sub-packages/bionemo-evo2/examples/configs/full_pretrain_shortphase_config.yaml \
+    --num-nodes=${SLURM_JOB_NUM_NODES} \
+    --devices=${SLURM_NTASKS_PER_NODE} \
+    --grad-acc-batches $GRAD_ACC_BATCHES \
+    --max-steps=$MAX_STEPS \
+    --seed $SEED \
+    ${EXTRA_ARGS} \
+    --no-wandb \
+    --lr $LR \
+    --wd $WD \
+    --min-lr $MIN_LR \
+    --warmup-steps $WU_STEPS \
+    --attention-dropout $ADO \
+    --hidden-dropout $HDO \
+    --limit-val-batches=20 \
+    --val-check-interval=${VAL_CHECK} \
+    --experiment-dir=/workspace/bionemo2/model/checkpoints/${EXPERIMENT_NAME} \
+    --seq-length=${SEQ_LEN} \
+    --tensor-parallel-size=${TP_SIZE} \
+    --context-parallel-size=${CP_SIZE} \
+    --pipeline-model-parallel-size=${PP_SIZE} \
+    --workers 8 \
+    --micro-batch-size=${MICRO_BATCH_SIZE} \
+    --model-size=${MODEL_SIZE}
+EOF
+srun \
+    --output ${RESULTS_PATH}/slurm-%j.out \
+    --error ${RESULTS_PATH}/error-%j.out \
+    --container-image=$IMAGE_NAME \
+    --container-mounts ${MOUNTS} \
```

Comment:
So yeah; this is not standard SLURM, this is [enroot](github.com/nvidia/enroot) -- I think providing some sample SLURM scripts is good, but I'm worried a lot of this is NVIDIA-specific.

Reply:
`--container-image` is enroot specific? What if you supplied a docker container as `IMAGE_NAME=[INSERT IMAGE NAME HERE]`?

Reply:
https://slurm.schedmd.com/containers.html interestingly the slurm docs talk about enroot

Reply:
I think this is addressed by the blurb @jwilber added:
> The scripts are configured for the [DGX cloud SLURM environment](https://docs.nvidia.com/dgx-cloud/slurm/latest/cluster-user-guide.html). While some settings are specific to our cluster resources, these scripts can be adapted for other HPC environments.